org.eclipse.lsp4j.websocket.jakarta/build.gradle

```diff
@@ -0,0 +1,24 @@
+/******************************************************************************
+ * Copyright (c) 2019 TypeFox and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ ******************************************************************************/
+
+ext.title = 'LSP4J WebSocket Jakarta'
+description = 'Jakarta WebSocket support for LSP4J'
+
+dependencies {
+	compile project(":org.eclipse.lsp4j.jsonrpc")
+	compile "jakarta.websocket:jakarta.websocket-api:2.0.0"
```

Comment:
here is should use a variable coming from the settings.gradle. it wasn't picking it when tried. Need to investigate more

Reply:
it was because the versions properties cannot contain a dot. so I replaced `websocket.jakarta` by `websocket_jakarta` and it is working fine.